QEfficient/transformers/pytorch_transforms.py

```diff
@@ -194,4 +203,11 @@ class CBTransform(KVCacheTransform):
         MixtralDecoderLayer: QeffMixtralDecoderLayer,
         # Starcoder2
         Starcoder2DecoderLayer: QEFFStarcoder2DecoderLayer,
+        # Codegen
+        CodeGenBlock: QeffCodeGenBlock,
+        # GPT2
```

Comment:
GPT2?

Reply:
GPT2 is not having separate decoderlayer implemented, its part of GPT2Block. I will remove the comment.